drivers/smbus/intel_pch_smbus.c

```diff
@@ -53,6 +54,21 @@ struct pch_data {
 
 	struct k_mutex mutex;
 	struct k_sem completion_sync;
+	const struct device *dev;
+
+	/* smbalert callback list */
+	sys_slist_t smbalert_cbs;
+	/* smbalert work */
+	struct k_work smb_alert_work;
+
+	/* Host Notify callback list */
+	sys_slist_t host_notify_cbs;
+	/* Host Notify work */
+	struct k_work host_notify_work;
+	/* Host Notify peripheral device address */
+	uint8_t notify_addr;
+	/* Host Notify data received */
+	uint16_t notify_data;
```

Comment:
Wrap in `ifdef CONFIG_SMBUS_INTEL_PCH_HOST_NOTIFY`?

Reply:
Added wrap and tests